### PR TITLE
updates for eslint cleanup

### PR DESCRIPTION
**Pull request by mikekidder**
Notes: Minor changes to code in async, real-world examples to clear the eslint errors.

### CODE REVIEW

File: src/utils/combineReducers.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function verifyStateShape(initialState, currentState) {
   var reducerKeys = Object.keys(currentState);
 
   if (reducerKeys.length === 0) {
-    console.error(
+    console.error( // eslint-disable-line no-console
```
Comment (hzoo) [inline src/utils/combineReducers.js]: maybe just disable the `no-console` rule?

Comment (mikekidder) [inline src/utils/combineReducers.js]: could if needed elsewhere. just the three lines needed it in one file.

File: examples/real-world/actions/index.js [hunk 2]
```diff
@@ -3,10 +3,9 @@ import { CALL_API, Schemas } from '../middleware/api';
 export const USER_REQUEST = 'USER_REQUEST';
 export const USER_SUCCESS = 'USER_SUCCESS';
 export const USER_FAILURE = 'USER_FAILURE';
-/**
- * Fetches a single user from Github API.
- * Relies on the custom API middleware defined in ../middleware/api.js.
- */
+
+// Fetches a single user from Github API.
```
Comment (gaearon) [inline examples/real-world/actions/index.js]: Doesn't ESLint like JSDoc?
That's just weird.

Comment (mikekidder) [inline examples/real-world/actions/index.js]: It does, just wanted the return variable and parameters passed as well.  Wasn't sure if you were really putting in JSDoc comments.  I can change back and add the missing parts.

Comment (gaearon) [inline examples/real-world/actions/index.js]: Ah, okay, I don't mind them being `//` in this case.

**Comment by gaearon** [comment]
Is linting no longer running as part of Travis build? Why did we not see these lint errors?

**Comment by mikekidder** [comment]
I looked at history of .travis.yml, don't believe linting was ever part of it?

**Comment by gaearon** [comment]
Lol, maybe so. Would you mind adding it?

**Comment by hzoo** [comment]
Might want to update to eslint 1.2.1 and babel-eslint to 4.0.10 as well (although that could be another PR)?

**Comment by gaearon** [comment]
@hzoo If you would like to submit a separate PR updating them, please feel free to! I haven't used `eslint` 1.0 yet.

**Comment by mikekidder** [comment]
How about @hzoo adds lint to Travis with that PR.  I am getting an error running "npm run lint" under windows, even for the master branch.  Throwing an exception in \eslint\lib\config-validator.js, so its going to need to be looked at some more before adding to Travis CI.

**Comment by mikekidder** [comment]
... yeah, its saying this:

 Configuration for rule "generator-star-spacing" is invalid:
        Value "[object Object]" must be an enum value.

**Comment by mikekidder** [comment]
Successful linting in Travis CI  :)

**Comment by mikekidder** [comment]
@hzoo I also have changes ready to go with the following npm updates:
babel-eslint 4.1.0, eslint 1.2.1, eslint-plugin-react 3.2.3, eslint-config-airbnb 0.0.8

If you are not working on a PR for it, I will submit.  Let me know!

**Comment by hzoo** [comment]
I haven't started it yet so go ahead!

**Pull request merged**